src/views/Home/Home.js

```diff
@@ -72,12 +72,15 @@ const Home = () => {
           </span>
         )}
         <br />
-        Every two weeks, the preferred project of the community will receive the
-        interest of{' '}
+        The previous winning project has earned{' '}
         {interest > 0 && (
-          <span className={classes.interestCountUp}>${interest}!</span>
+          <span className={classes.interestCountUp}>${interest} DAI!</span>
         )}
-        {interest === 0 && <span>{<EllipsisLoader />}</span>}
+        {interest === 0 && <span>{<EllipsisLoader />}</span>} so far this
+        period.
+        <br />
+        At this rate the project will earn <strong>...TODO...</strong> DAI by
```

Comment:
I thought adding something like this would be nice, basically at the beginning of the iteration  when there is very little interest it would be good to give a forecast/prediction of the amount that will be raised.

Reply:
Yes, this statistic is way better than the other. The other accumulator one should never be used in my opinion, (at least until the pool is massive). 

Still I think we should leave this one out too, and only focus on the metric of pool size at the beginning during the growth phase. We can maybe list the interest rate we expect (even pull this from Aave), and users can work it out if they really like.